### ISSUE TITLE
 dependadotnet cli help 

### ISSUE THREAD

**Issue by motowilliams** [dotnet/core#5582]
` dependadotnet /?` or `dependadotnet --help` should show the usage text. Instead of tried to append that string to the current (execution) directory.

**Comment by richlander** [comment]
Good call. Feel free to contribute it, or I will add it in a few days.
